apps/saclock/app.js

```diff
@@ -67,13 +71,18 @@ let clock = new (require("ClockFace"))({
     }
 
     buf.setColor(g.theme.bg).fillCircle(c, c, r2); // clear inside
-    buf.setColor(g.theme.fg);
+    buf.setColor(this.multicol ? g.theme.fg2 : g.theme.fg);
     drawHand(hl, hr); // hour hand
+    buf.setColor(g.theme.fg);
     drawHand(ml, mr); // minute hand
     buf.fillCircle(c, c, Math.floor(hw)); // hands joiner
+    if (!this.multicol && !g.theme.dark) {
+        // apparently 1-bit g.drawImage swaps fg/bg colors for light themes?
```

Comment:
I'm not sure why we need this to prevent inverted colors for monochrome on light theme. But it works now...